index.js

```diff
@@ -49,7 +49,10 @@ var Parser = require('i18next-scanner').Parser;
 var parser = new Parser({
   keepRemoved: false,
   keySeparator: false,
-  sort: true
+  sort: true,
+  resource: {
+    lineEnding: '\n'
```

Comment:
This is the default: https://github.com/i18next/i18next-scanner#resource

```

        // Normalize line endings to '\r\n', '\r', '\n', or 'auto' for the current operating system. Defaults to '\n'.
        // Aliases: 'CRLF', 'CR', 'LF', 'crlf', 'cr', 'lf'
        lineEnding: '\n'
```

Doesn't harm being explicit about it though.